### PR TITLE
1.7.4 is ready for translation

### PR DESCRIPTION
**Pull request by LouiseBonnard**

### CODE REVIEW

**Review by ttoine — CHANGES_REQUESTED** [review]
--- we decided not to close the previous branches anymore in order to avoid ---
il manque quelque chose, on dirait

File: news/_posts/translation/2018-06-22-174-translations.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+<img width="990" src="/assets/images/2018/06/Translation_174.jpg">
+
+
+### Handling the Crowdin catalog
+
+Until then, our usage consisted in adding a new branch, corresponding to the new minor version, and this new branch should contain all the strings from the previous branch (the previous version, you got it right) plus the new ones (relating to the new version, you got it right twice, you smarties).
+
+However, we later realized that keeping only the latest minor version of PrestaShop active on Crowdin was not so good an idea. Indeed, --- we decided not to close the previous branches anymore in order to avoid ---
```
Comment (eternoendless) [inline news/_posts/translation/2018-06-22-174-translations.md]: I think this whole section can be deleted as this hasn't been addressed yet

File: news/_posts/translation/2018-06-22-174-translations.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+
+### About the multiple generations of the 1.7.4 catalog
+
+And that know, let’s explain why we had to regenerate the 1.7.4 catalog many times. We are very sorry for the relentless translators who have had to do the job twice, be sure the problem is solved. What happened is that we first had a bug in the library we use to extract strings, then we figured out we had deleted strings we thought were useless that actually were not that useless… so we put them back in the code and everything went back in order. Phew!
```
Comment (eternoendless) [inline news/_posts/translation/2018-06-22-174-translations.md]: And that said, you may have noticed that we had to regenerate the 1.7.4 catalog many times. We are very sorry for the relentless translators who have had to do the job twice, be rest assured, the problem is now solved. What happened is that we first had a bug in the library we use to extract strings (it wasn't able to extract some of them), then we realized we had deleted some strings we thought we didn't need but actually we did… so we put them back in the code and everything went back in order. Phew!

File: news/_posts/translation/2018-06-22-174-translations.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+
+### Natural fall of all translation levels
+
+Improving PrestaShop naturally leads to new wording. As a consequence, the 1.7.4 branch contains more strings than the 1.7.3. So all the languages that were fully translated and validated will fall under 100%. No need to panic, it is just logic. Once these few new strings translated and approved, rates of completion will get back to normal.
+
+Of course, as the context is not fully available because PrestaShop 1.7.4 is not out for now (but will soon be), some of these strings may appear a bit unclear. But ask if you need explanations, we will do our best to answer your comments and issues - and do not forget the coming soon release will allow you to check the results and make them match a more faithful translation.
```
Comment (eternoendless) [inline news/_posts/translation/2018-06-22-174-translations.md:36]: They can actually check it out if they download the beta

File: news/_posts/translation/2018-06-22-174-translations.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+
+### Go for it!
+
+All this to say new strings have been added to our projects available in Crowdin. Modifications are more than allowed, it will help us having [PrestaShop 1.7.4](https://crowdin.com/project/prestashop-official) as much ready as possible for all!
```
Comment (eternoendless) [inline news/_posts/translation/2018-06-22-174-translations.md:41]: > Modifications are more than allowed

"Modifications are more than _welcome_"

> as much ready as possible for all!

There's a problem there, I don't think you can say "much ready" (unless you're doge: much ready, such finished).

I believe "As ready as possible" would be fine 🙂

**Pull request merged**